src/OmniSharp.Abstractions/Models/v1/FilesChanged/ChangeTypeConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace OmniSharp.Models.FilesChanged
+{
+    class FileChangeTypeConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType) => objectType == typeof(FileChangeType);
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            switch (reader.Value as string)
+            {
+                case "Change":
+                    return FileChangeType.Change;
+                case "Create":
+                    return FileChangeType.Create;
+                case "Delete":
+                    return FileChangeType.Delete;
+                default:
+                    return null;
+            }
+        }
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer) => throw new NotImplementedException();
```

Comment:
If we keep this we should override `CanWrite` and return false, would hate to have this throw unexpectedly.